spec/steps/execute.fmf

```diff
@@ -11,29 +11,41 @@ description: |
     in the following format::
 
         /test/one:
-            result: TEST_RESULT
+            result: OUTCOME
             log: PATH
 
         /test/two:
-            result: TEST_RESULT
+            result: OUTCOME
             log: PATH
 
-    Where ``TEST_RESULT`` is the outcome of the test. It can have
-    these values:
+    Where ``OUTCOME`` is the result of the test execution. It can
+    have the following values:
 
-        * info - test finished and produced only information
-          message
-        * passed - test finished and passed
-        * failed - test finished and failed
-        * error - a problem encountered during test execution,
-          treated as failed
+        pass
+            Test execution successfully finished and passed.
+        info
+            Test finished but only produced an informational
+            message. Represents a soft pass, used for skipped
+            tests and for tests with the :ref:`/spec/tests/result`
+            attribute set to *ignore*. Automation should treat
+            this as a passed test.
+        warn
+            A problem appeared during test execution which does
```

Comment:
note about how should the automation treat this. I believe it should be a failed test also right?

Reply:
Yeah, you're right. By default it should treated as fail. Updated the spec. In the future we might also consider to make this configurable: Use would choose how a `warn` status should be handled.